# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/profiler/core/platform.h
# Commit: 6b399daabb18
# Full Hash: 6b399daabb187b957004361b4e53c43afc61e420
# Author: Nazım Can Altınova <canaltinova@gmail.com>
# Date: 2024-11-26 09:36:10
# Regressor Bug: 1634785
# File Overlap Count: 1
# Description:
#   Bug 1634785 - Rename BaseProfilerSharedLibraries.h to SharedLibraries.h r=mstange,profiler-reviewers
#   
#   Now that we only have one implementation of this code, we don't have to
#   differentiate it with "BaseProfiler".
#   
# ==============================================================================

diff -r 8ad3b7e6de23 -r 6b399daabb18 tools/profiler/core/platform.h
--- a/tools/profiler/core/platform.h	Mon Nov 25 22:09:05 2024 +0000
+++ b/tools/profiler/core/platform.h	Mon Nov 25 22:09:05 2024 +0000
@@ -44,7 +44,7 @@
 #include "mozilla/UniquePtr.h"
 #include "mozilla/Vector.h"
 #include "nsString.h"
-#include "BaseProfilerSharedLibraries.h"
+#include "SharedLibraries.h"
 
 #include <cstddef>
 #include <cstdint>